march_gazebo_plugins/CMakeLists.txt

```
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 2.8FATAL_ERROR)
+project(march_gazebo_plugins)
+
+# Load catkin and all dependencies required for this package
+find_package(catkin REQUIRED COMPONENTS
+        march_shared_resources
+        )
+find_package(gazebo REQUIRED)
+
+catkin_package()
+
+include_directories(
+    ${catkin_INCLUDE_DIRS}
+    ${GAZEBO_INCLUDE_DIRS}
+)
+
+add_library(${PROJECT_NAME} src/com_controller.cpp)
+target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES} ${GAZEBO_LIBRmaARIES})
```

I don't think the `${GAZEBO_LIBRmaARIES}` contains anything, so it should be safe to remove :upside_down_face: 
```suggestion
target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES})
```